### ISSUE TITLE
Get accessibility tests passing

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#1302]
### User Story or Problem Statement

As a developer I need our accessibility tests to pass so that when we see failures, we can take action to address them.

As a CMS user I need our CMS to meet accessibility requirements so that I can use the site without restrictions.

This is an epic, so that each failure can get its own issue.

### Affected users and stakeholders

* CMS_editors_of_X_product
* Developers of Y teams

### Assumptions
* composer va:test:accessibility adequately tests the accessibility of the CMS.

(How will these assumptions be validated?)

## Acceptance Criteria
* composer va:test:accessibility  passes.

## Possible tickets to create for this epic

- [ ] Spike task, with some of the following attributes:
* Identify the baseline against which we will measure improvements
* Ideation (sketching, cross-displinary meeting of minds around how the problem may be solved)
* Analysis of the drupal landscape
* Possible implementation(s)
* Proof of concept or prototype
* UX writing discovery

## Labels for  this issue
- [/] Issue type (red) (defaults to Epic)
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [ ] CMS objective (orange)
- [ ] CMS-supported product (black)
